UK/Data/Settings/Lists_SMR.txt

```diff
@@ -205,6 +206,7 @@ m_Column:ASSR:5:1:60:9000:9022:0::UK Controller Plugin:UK Controller Plugin:0:0.
 m_Column:RLS:3:1:124:9012:9015:1:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
 m_Column::6:0:125:9014:9014:1:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
 m_Column:PRE:3:1:127:9017:9018:1:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
+m_Column:QSY:7:0:107:0:0:0:UK Controller Plugin:::1:0.0
```

Comment:
Disagree with this change. The next frequency is only applicable to TWR controller as it's the next airborne frequency. Having it only in the take-off lists encourages controllers to use the GND STS field correctly.

Reply:
I see, my idea was that someone might still use the departure list so I wanted to add the functionality there as well.